master-firmware/tests/test_strategy.cpp

```diff
@@ -176,3 +185,39 @@ TEST(Strategy, CanBuildTower)
     CHECK_TRUE(len > 0);
     CHECK_TRUE(goal.is_reached(state));
 }
+
+TEST(Strategy, CanBuildSecondTower)
+{
+    const int max_path_len = 10;
```

Comment:
[nitpick] Those tests (CanDoWhatever) have a lot in common. If it was for something longer-lived than the strategy I would suggest refactoring them.